scripts/exregional_nexus_emission.sh

```diff
@@ -0,0 +1,323 @@
+    ln_vrfy -sf "${COMINext}/GFS_SFC" .
+    USE_GFS_SFC="TRUE"
+  fi
+fi
+
+#
+#-----------------------------------------------------------------------
+#
+# Copy the NEXUS config files to the tmp directory  
+#
+#-----------------------------------------------------------------------
+#
+cp_vrfy ${EXECdir}/nexus ${DATA}
+cp_vrfy ${NEXUS_FIX_DIR}/${NEXUS_GRID_FN} ${DATA}/grid_spec.nc
+
+
+if [ "${USE_GFS_SFC}" = "TRUE" ]; then
+    cp_vrfy ${ARL_NEXUS_DIR}/config/cmaq_gfs_megan/*.rc ${DATA}
+else
+    cp_vrfy ${ARL_NEXUS_DIR}/config/cmaq/*.rc ${DATA}
+fi
+#
+#-----------------------------------------------------------------------
+#
+# Get the starting and ending year, month, day, and hour of the emission
+# time series.
+#
+#-----------------------------------------------------------------------
+#
+mm="${PDY:4:2}"
+dd="${PDY:6:2}"
+hh="${cyc}"
+yyyymmdd="${PDY}"
+
+NUM_SPLIT_NEXUS=$( printf "%02d" ${NUM_SPLIT_NEXUS} )
+if [ "${FCST_LEN_HRS}" = "-1" ]; then
+  for i_cdate in "${!ALL_CDATES[@]}"; do
+    if [ "${ALL_CDATES[$i_cdate]}" = "${PDY}${cyc}" ]; then
+      FCST_LEN_HRS="${FCST_LEN_CYCL[$i_cdate]}"
+      break
+    fi      
+  done
+fi
+
+if [ "${NUM_SPLIT_NEXUS}" = "01" ]; then
+  start_date=$( $DATE_UTIL --utc --date "${yyyymmdd} ${hh} UTC" "+%Y%m%d%H" )
+  end_date=$( $DATE_UTIL --utc --date "${yyyymmdd} ${hh} UTC + ${FCST_LEN_HRS} hours" "+%Y%m%d%H" )
+else
+  len_per_split=$(( FCST_LEN_HRS / NUM_SPLIT_NEXUS  ))
+  nsptp=$(( nspt+1 ))
+
+  # Compute start and end dates for nexus split option
+  start_del_hr=$(( len_per_split * nspt ))
+  start_date=$( $DATE_UTIL --utc --date "${yyyymmdd} ${hh} UTC + ${start_del_hr} hours " "+%Y%m%d%H" )
+  if [ "${nsptp}" = "${NUM_SPLIT_NEXUS}" ];then
+    end_date=$( $DATE_UTIL --utc --date "${yyyymmdd} ${hh} UTC + ${FCST_LEN_HRS} hours" "+%Y%m%d%H" )
+  else
+    end_del_hr=$(( len_per_split * nsptp ))
+    end_date=$( $DATE_UTIL --utc --date "${yyyymmdd} ${hh} UTC + ${end_del_hr} hours" "+%Y%m%d%H" )
+  fi
+fi
+#
+#######################################################################
+# This will be the section to set the datasets used in $workdir/NEXUS_Config.rc 
+# All Datasets in that file need to be placed here as it will link the files 
+# necessary to that folder.  In the future this will be done by a get_nexus_input 
+# script
+NEI2016="TRUE"
+TIMEZONES="TRUE"
+CEDS="TRUE"
+HTAP2010="TRUE"
+OMIHTAP="TRUE"
+MASKS="TRUE"
+NOAAGMD="TRUE"
+SOA="TRUE"
+EDGAR="TRUE"
+MEGAN="TRUE"
+MODIS_XLAI="TRUE"
+OLSON_MAP="TRUE"
+Yuan_XLAI="TRUE"
+GEOS="TRUE"
+AnnualScalar="TRUE"
+
+NEXUS_INPUT_BASE_DIR=${NEXUS_INPUT_DIR}
+########################################################################
+
+#
+#----------------------------------------------------------------------
+# 
+# modify time configuration file
+#
+cp_vrfy ${ARL_NEXUS_DIR}/utils/python/nexus_time_parser.py .
+echo ${start_date} ${end_date} # ${cyc}
+./nexus_time_parser.py -f ${DATA}/HEMCO_sa_Time.rc -s $start_date -e $end_date
```

Comment:
Should we use `python3 $DIR/nexus_time_parser.py` to be consistent with other tasks like get_extrn_ics/lbcs?